lib/huginn_agent/templates/newagent/newagent.gemspec.tt

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
 
   spec.files         = Dir['LICENSE.txt', 'lib/**/*']
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = Dir['spec/**/*.rb']
+  spec.test_files    = Dir['spec/{[!huginn/]}**/*.rb']
```

Comment:
ouch, you're right. Sorry for not testing well enough, I just glanced at the gems size and it shrank (too far as it excluded the proper specs also)